util/image_viewer/slip_viewer/frame.py

```diff
@@ -524,70 +524,7 @@ def get_key(self, file_name_or_data):
             return super().get_key(file_name_or_data)
 
     def update_settings(self, layout=True):
-        # XXX The zoom level from the settings panel are not taken into
```

Comment:
@rjgildea improving his -ve line count, approve! 

This aliasing did completely confuse me while I was trying to change the default, so appreciate it being removed. 

That's a couple of haunted trees felled anyway.